packages/alea-frontend/pages/api/comment-utils.ts

```diff
@@ -162,9 +162,9 @@ export async function getExistingPointsDontEnd(
   return { existing: existingGrant[0], error: undefined };
 }
 
-export function checkIfPostOrSetError(req, res) {
-  if (req.method !== 'POST') {
-    res.status(405).send({ message: 'Only POST requests allowed' });
+export function checkIfTypeOrSetError(req, res, type: 'POST' | 'DELETE' = 'POST') {
```

Comment:
Don't remove `checkIfPostOrSetError`. Rewrite it like so:
```
export function checkIfPostOrSetError(req, res) {
return checkIfTypeOrSetError(req, res, 'POST');
}
```
Similary create the function `checkIfDeleteOrSetError` using `checkIfTypeOrSetError`

The reason for this:
`checkIfTypeOrSetError` is confusing to anyone who reads the code. They wont understand what is 'Type' and why 'POST' is default